Src/Witsml/ServiceReference/OptionsIn.cs

```diff
@@ -28,9 +34,22 @@ public string GetKeywords()
             {
                 keywords.Add($"requestObjectSelectionCapability=true");
             }
+            if (!string.IsNullOrEmpty(OptionsInString))
```

Comment:
In the validation, it only checks that it matches the regex if it's not null or empty. So an empty string or null will not throw. If I add an empty string to the keywords-list, it will be joined on afterwards, resulting a double ;; in the final keywords string.